client/me/purchases/manage-purchase/index.jsx

```diff
@@ -157,10 +157,59 @@ const ManagePurchase = React.createClass( {
 		);
 	},
 
+	getExpiringText( purchase ) {
+		if ( purchase.expiryStatus === 'manualRenew' ) {
+			return (
+				this.translate( '%(purchaseName)s will expire and be removed from your site %(expiry)s. ',
+					{
+						args: {
+							purchaseName: getName( purchase ),
+							expiry: this.moment( purchase.expiryMoment ).fromNow()
+						}
+					}
+				) +
+				' ' +
+				this.translate( 'Please, add a credit card if you want it to autorenew.' )
+			);
+		}
+
+		if ( purchase.expiryStatus === 'manualRenew' ) {
+			return this.translate( '%(purchaseName)s will expire and be removed from your site %(expiry)s. ',
+					{
+						args: {
+							purchaseName: getName( purchase ),
+							expiry: this.moment( purchase.expiryMoment ).fromNow()
+						}
+					}
+				);
+		}
+		if ( isMonthly( purchase.productSlug ) ) {
+			const expiryMoment = this.moment( purchase.expiryMoment );
+			const daysToExpiry = this.moment( expiryMoment.diff( this.moment() ) ).format( 'D' );
+
+			return this.translate( '%(purchaseName)s will expire and be removed from your site %(expiry)s days. ',
```

Comment:
Alternate string suggestion: 
* `translate( '%(purchaseName)s will expire and be removed from your site %(expiry)s.' )`&mdash; translations: **20**. *ES Score: 8.72*

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/plans-improved-renew-warnings)